data_collection/gazette/spiders/pr_cianorte.py

```diff
@@ -0,0 +1,31 @@
+from dateparser import parse
+from scrapy.http import Request
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+class PrCianorteSpider(BaseGazetteSpider):
```

Comment:
We should have a `start_date` attribute here. This will be used in production to capture gazettes in the last few days while not making unnecessary requests. In this spider, since we go back in time by paginating, you could verify if the gazette date you are extracting is before `self.start_date` and stop the crawl right there if you reach this point.

If in doubt, check some spiders for capitals, most of them already have `start_date`.

Reply:
:+1: this is an issue that needs to be solved.